SalesforceCommerceCloud/pwa-kit
# Fix: a11y - Add accessibility label to edit/remove buttons on checkout page (@W-12627228@)

# Description

This PR adds descriptive accessibility label to the edit/remove buttons on the Checkout page, specifically within the shipping addresses section so you know which edit/remove button is associated to which address.



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

1. `git checkout ju/a11y-btn-label`
2. start the retail react app
3. add a product to cart and proceed to checkout
4. Move onto the shipping addresses section
5. use screen reader to read edit/remove button
6. Observe that the screen reader reads out the accessibility label: `Edit {address}` or `Remove {address}` 

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
